internal/testharness/pty/analyzer/types.go

```
@@ -109,6 +134,218 @@ func AfterChunk(index int) ResizePlacement {
+func (a *CaptureAssembler) tailExcerpt(extra []byte) []byte {
+	size := min(int(a.offset)+len(extra), evidenceExcerptSize)
+	tail := make([]byte, size)
+	fromExtra := min(len(extra), size)
+	if fromExtra > 0 {
+		copy(tail[size-fromExtra:], extra[len(extra)-fromExtra:])
+	}
+	a.copyEvidenceTail(tail[:size-fromExtra])
+	return tail
+}
+
+func (a *CaptureAssembler) evidenceBytes() []byte {
```

golangci-lint (2.12.2)

[error] func (*CaptureAssembler).evidenceBytes is unused

(unused)